frame/ibc/src/lib.rs

```diff
@@ -209,13 +198,11 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[allow(clippy::disallowed_types)]
-	/// connection_id => ConnectionEnd
-	pub type Connections<T: Config> =
-		CountedStorageMap<_, Blake2_128Concat, Vec<u8>, Vec<u8>, ValueQuery>;
+	pub type ChannelCounter<T: Config> = StorageValue<_, u32, ValueQuery>;
 
 	#[pallet::storage]
 	#[allow(clippy::disallowed_types)]
-	pub type ChannelCounter<T: Config> = StorageValue<_, u32, ValueQuery>;
+	pub type PacketCounter<T: Config> = StorageValue<_, u32, ValueQuery>;
```

Comment:
This would probably not be needed, since we'll rewrite the `on_finalize` benchmark